packages/components/src/color-palette/style.scss

```diff
@@ -47,6 +47,8 @@ $color-palette-circle-spacing: 14px;
 	&.is-active {
 		box-shadow: inset 0 0 0 4px;
 		border: $border-width solid $dark-gray-400;
+		position: relative;
+		z-index: 1;
```

Comment:
@jasmussen - should there be special handling for `z-index` used?

Reply:
My impression is that's always good. But I'm honestly not confident, sometimes a z index of 1 is just to elevate a child in a context that isn't subject to the mess it can otherwise create. I know @aduth has thoughts.

Reply:
I'll leave it to him then :)

Reply:
My impression has been to always use `_z-index` function for any `z-index`, with the mapping serving as a centralized authority to serve in proactively avoiding aggressively and endlessly increasing a `z-index` value to "combat" others.

https://github.com/WordPress/gutenberg/blob/master/assets/stylesheets/_z-index.scss

But:

1. For a case like this, where the value is `1`, perhaps it's reasonable to make exceptions
2. In any case, we don't codify this guideline one way or the other. Whatever we decide, it ought to be written down, e.g. `coding-guidelines.md` document